src/react-apps/applications/ux-editor/src/App.tsx

```diff
@@ -73,6 +74,10 @@ export class App extends React.Component<IAppComponentProps, IAppCompoentState>
       manageServiceConfigurationActionDispatcher.fetchJsonFile(
         `${altinnWindow.location.origin}/runtime/api/resource/${servicePath}/ServiceConfigurations.json`);
 
+      // Fetch current workflow state
+      WorkflowActionDispatcher.getCurrentState(
+        `${window.location.origin}/runtime/${org}/${service}/${instanceId}/GetCurrentState?reporteeId=${reportee}`);
```

Comment:
use `altinnWindow` and `servicePath` to be consistent with the other api calls

Reply:
Fixed.